cfgov/unprocessed/js/routes/on-demand/simple-chart/simple-chart.js

```diff
@@ -117,6 +118,40 @@ function makeChartOptions( data, dataAttributes ) {
     };
   }
 
+  if (projectedMonths) {
+  // Convert the number of projected months into a timestamp
+  const lastChartDate = defaultObj.series[0].data.at(-1).x;
+
+  // Convert lastChartDate from months to milliseconds for Epoch format
+  const convertedProjectedDate = lastChartDate - projectedMonths * 30 * 24 * 60 * 60 * 1000;
+  const projectedDate = getProjectedDate( convertedProjectedDate );
+
+  // Add a vertical line and some explanatory text at the starting
+  // point of the projected data
+  defaultObj.xAxis.plotLines = [ {
+    value: projectedDate.timestamp,
+    label: {
+      text: `Values after ${projectedDate.humanFriendly} are projected`,
+      rotation: 0,
+      useHTML: true,
```

Comment:
I *think* `useHTML` can be removed because we're not using any HTML in the label text.